features/networking/service.feature

```diff
@@ -153,3 +153,438 @@ Feature: Service related networking scenarios
     Then the step should succeed
     And the output should contain:
       | 8080 |
+
+  # @author weliang@redhat.com
+  # @case_id OCP-24668
+  @admin
+  Scenario: externalIP defined in service but no sepc.externalIP defined	
+    Given I have a project 
+    And evaluation of `project.name` is stored in the :proj1 clipboard
+    
+    # Create a service with a externalIP in :proj1
+    When I run the :create client command with:
+      | f | https://raw.githubusercontent.com/openshift-qe/v3-testfiles/master/networking/externalip_service1.json | 
+      | n | <%= cb.proj1 %>                                                                                        |
+    Then the output should contain "externalIPs have been disabled" 
```

Comment:
I don't see any admin operations performed in this senario. You can simplify as @anuragthehatter suggested to:
```
Given I have a project
When I run the :create client command with:
  | f | https://raw.githubusercontent.com/openshift-qe/v3-testfiles/master/networking/externalip_service1.json | 
Then the step should fail
And the output should contain "externalIPs have been disabled" 
```

Reply:
Remove @admin tag in test case OCP-24668
Add @@destructive tag in five test cases